src/views/network/view_session.cpp

```diff
@@ -44,6 +45,18 @@ namespace big
 		ImGui::Checkbox("Disable Filter", &g->session.disable_chat_filter);
 		ImGui::Checkbox("Log Chat Messages", &g->session.log_chat_messages);
 		ImGui::Checkbox("Log Text Messages", &g->session.log_text_messages);
+		static char msg[256];
+		ImGui::InputText("##message", msg, 256);
```

Comment:
Use `sizeof(msg)`, it will prevent bugs if the size of the message is only changed in one location.
Unlikely in this use case but it's a good habit to have.